dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsDataModel.ts

```diff
@@ -0,0 +1,642 @@
+            label: "Substantial Contribution to Protection and Restoration of Biodiversity and Ecosystems",
+            description:
+              "The value in percentage of the contribution to protection and restoration of biodiversity and ecosystems",
+            unit: "Percentage",
+            component: "PercentageFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+          },
+
+          {
+            name: "alignedActivities",
+            label: "Aligned OpEx per Activity",
+            description:
+              "Absolute value and share of the opex per activity that is taxonomy-aligned, i.e., generated by an eligible economic activity that is making a substantial contribution to at least one of the climate and environmental objectives, while also doing no significant harm to the remaining objectives and meeting minimum standards on human rights and labour standards",
+            unit: "",
+            component: "AlignedActivitiesFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+          },
+          {
+            name: "nonAlignedShare",
+            label: "Total Non-Aligned OpEx",
+            description:
+              "Absolute value and share of the OpEx that is associated with not taxonomy-aligned activities. i.e., for an eligible economic activity that is making a substantial contribution to at least one of the climate and environmental objectives, while also doing no significant harm to the remaining objectives and meeting minimum standards on human rights and labour standards",
+            unit: "(Percentage)",
+            component: "FinancialShareFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+          },
+          {
+            name: "nonAlignedActivities",
+            label: "Non-Aligned OpEx per Activity",
+            description:
+              "Absolute value and share of the opex per activity that is not taxonomy-aligned, i.e., not generated by an eligible economic activity that is making a substantial contribution to at least one of the climate and environmental objectives, while also doing no significant harm to the remaining objectives and meeting minimum standards on human rights and labour standards",
+            unit: "",
+            component: "NonAlignedActivitiesFormField",
+            evidenceDesired: false,
+            required: false,
+            showIf: (): boolean => true,
+          },
+          {
+            name: "nonEligibleShare",
+            label: "Total Non-Eligible OpEx",
+            description:
+              "Absolute value and share of the OpEx that is not part of a plan to meet the taxonomy criteria for substantial contribution to climate change adaptation and relevant DNSH criteria",
+            unit: "(Percentage)",
```

Comment:
What is the difference between `(Percentage)` and `Percentage` as the unit?

Reply:
What is the unit field for used anways? I can't seem to find that right away

Reply:
I would assume that it should be Percentage and not (Percentage).
Unit entry is used to give information in which unit the value is given (could be a currency or GW or kg).  In case of eu taxonomy it might be not really used, but in case of SFDR we used it to handle all unit related stuff.
Unit is then automatically generated by the devtools

Reply:
I still can find them. Did your push your changes? In the display file they are gone.

Reply:
Now also in this file. I missed some of them